spec/mailers/user_mailer_spec.rb

```diff
@@ -76,6 +76,15 @@
         "[Reservations] #{@res.equipment_model.name} Request Approved")
     end
 
+    it 'sends expired request notifications' do
+      @res.update_attributes(status: 'denied',
+                             flags: (Reservation::FLAGS[:request] |
```

Comment:
This is so sweet :smile:. Can we update the [wiki page](https://github.com/YaleSTC/reservations/wiki/Reservation-Flags) with how you'd assign multiple flags simultaneously like this as an alternative to the `Reservation#flag` method?